src/sentry/api/bases/organization_events.py

```diff
@@ -200,21 +209,36 @@ def build_cursor_link(self, request, name, cursor):
         else:
             base_url = base_url + "?"
 
-        return LINK_HEADER.format(
-            uri=base_url,
-            cursor=str(cursor),
-            name=name,
-            has_results="true" if bool(cursor) else "false",
+        return cast(
```

Comment:
I took a look at this and why it's required and I wonder if whenever we type out api/base, we actually want to change this in a function with typed args: `def get_link_header(uri: str, cursor: Cursor, name: str, has_results: bool)`

right now I think it may be slightly more typesafe to cast LINK_HEADER to str instead of `LINK_HEADER.format(..)`